checks/evaluation/dependency_update_tool.go

```diff
@@ -56,12 +56,16 @@ func DependencyUpdateTool(name string, dl checker.DetailLogger,
 
 	// Note: only one file per tool is present,
 	// so we do not iterate thru all entries.
-	dl.Info(&checker.LogMessage{
-		Path:   r.Tools[0].File.Path,
-		Type:   r.Tools[0].File.Type,
-		Offset: r.Tools[0].File.Offset,
-		Text:   fmt.Sprintf("%s detected", r.Tools[0].Name),
-	})
+	// Modified by AidenW on 05/31/2022: now Tool.File is a type of []File,
```

Comment:
Update this comment to just be `// Iterate over all the files, since a Tool can contain multiple files.`
People can use Git history if they want to know who/why it was changed.